src/components/ModalContent/ModalContent.stories.jsx

```diff
@@ -0,0 +1,12 @@
+// react libraries
+import * as React from 'react';
+
+// third-party libraries
+import { storiesOf } from '@storybook/react';
+
+// components
+import { ModalContent } from './index';
+
+
+storiesOf('ModalContent', module)
+  .add('click closed modal button', () => <ModalContent />);
```

Comment:
????

Reply:
I don't understand what is being done here. Could you please explain why we are writing stories for Modal content and how modal content is a reusable component?

Reply:
It is not needed. I will remove that.